x-pack/plugins/security_solution/public/management/components/endpoint_responder/get_running_processes_action.tsx

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useEffect, useMemo } from 'react';
+import styled from 'styled-components';
+import { EuiBasicTable } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { ActionDetails, RunningProcessesEntry } from '../../../../common/endpoint/types';
+import { useGetActionDetails } from '../../hooks/endpoint/use_get_action_details';
+import { EndpointCommandDefinitionMeta } from './types';
+import { CommandExecutionComponentProps } from '../console/types';
+import { useSendGetEndpointRunningProcessesRequest } from '../../hooks/endpoint/use_send_get_endpoint_running_processes_request';
+
+// @ts-expect-error TS2769
+const StyledEuiBasicTable = styled(EuiBasicTable)`
+  .euiTable {
```

Comment:
Is not as easy as I thought since we are using an `EuiBasicTable` and not an `EuiTable`. I will change that rule by using the `table` tag instead so we not depend on an eui class